packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherTrigger.tsx

```diff
@@ -33,6 +33,9 @@ export const OrganizationSwitcherTrigger = withAvatarShimmer(
         colorScheme='neutral'
         sx={[t => ({ minHeight: 0, padding: `0 ${t.space.$2} 0 0`, position: 'relative' }), sx]}
         ref={ref}
+        aria-label='organization-switcher-trigger'
+        aria-expanded={props.isOpen}
+        aria-haspopup='dialog'
```

Comment:
The button will also need an `aria-controls` tied to the popover contents

Reply:
this is inside `...rest`

Reply:
When inspecting the current components with dev tools I didn't see this. Can you double check that those are present?

Reply:
It's the last one

<img width="545" alt="image" src="https://github.com/clerkinc/javascript/assets/19269911/7166f180-1ca0-48a1-9717-fa2393bb1f2d">

Popover with the corresponding id
<img width="555" alt="image" src="https://github.com/clerkinc/javascript/assets/19269911/92861653-9139-4efe-9c68-4b4b4ebfd489">

Reply:
Ok 👍